service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java

```diff
@@ -601,8 +601,8 @@ public boolean checkIfTariffExists(AddNewTariffDto addNewTariffDto) {
     @Override
     public void setTariffLimitByAmountOfBags(Long tariffId, EditAmountOfBagDto dto) {
         TariffsInfo tariffsInfo = tryToFindTariffById(tariffId);
-        tariffsInfo.setMaxAmountOfBigBags(dto.getMaxAmountOfBigBags());
-        tariffsInfo.setMinAmountOfBigBags(dto.getMinAmountOfBigBags());
+        tariffsInfo.setMinQuantity(dto.getMaxAmountOfBigBags());
```

Comment:
you set `maxAmountOfBigBags` into `setMinQuantity`

Reply:
Is this logic covered by the test?
If it is the test should fail

Reply:
done